connect-example/src/main/java/com/stripe/android/connect/example/ui/accountloader/AccountLoaderViewModel.kt

```diff
@@ -25,7 +26,12 @@ class MainViewModel @Inject constructor(
     val state: StateFlow<MainState> = _state.asStateFlow()
 
     init {
-        fetchAccounts()
+        if (embeddedComponentService.accounts.value == null) {
+            _state.update { it.copy(isLoading = true) }
+            fetchAccounts()
+        } else {
+            _state.update { it.copy(isLoading = false) }
+        }
```

Comment:
(discussed IRL) Could we consolidate the state update logic (especially `isLoading`) in `fetchAccounts()` instead?